src/linux/socket/wg_socket.rs

```diff
@@ -132,7 +123,7 @@ impl WgSocket {
     pub fn set_device(&mut self, device: set::Device) -> Result<(), SetDeviceError> {
         for nl_message in create_set_device_messages(device, self.family_id)? {
             self.sock.send(nl_message)?;
-            self.sock.recv::<Nlmsg, Genlmsghdr<CtrlCmd, CtrlAttr>>()?;
+            self.sock.recv()?;
```

Comment:
Do you recall why the turbofish operator here was removed? Was there an compile error, runtime error, or is this just general code cleanup? If there was a compile or runtime error, I'd be curious to investigate a bit more to make sure the change here is expected.